tests/test_ocr.py

```diff
@@ -484,21 +484,40 @@ def test_ocr_eq_shouldnt_match(a, b):
     assert not stbt.ocr_eq(b, a)  # pylint:disable=arguments-out-of-order
 
 
+@contextmanager
+def temporary_ocr_eq_replacements():
+    orig = stbt.ocr_eq.replacements.copy()
+    try:
+        yield
+    finally:
+        stbt.ocr_eq.replacements = orig.copy()
```

Comment:
Don't need two copies here

```suggestion
        stbt.ocr_eq.replacements = orig
```